travispy/entities/hook.py

```diff
@@ -30,3 +31,29 @@ class Hook(Entity):
         'private',
         'admin',
     ]
+
+    def _set_hook(self, flag):
```

Comment:
Sorry, I missed that you specified putting these methods into the `Repo` entity.  Now, reviewing https://github.com/travis-ci/travis.rb/blob/dcc9f20535c811068c4ff9788ae9bd026a116351/lib/travis/client/repository.rb#L140-L151, I do concur that it makes more sense to put it into `Repo`.  Does it make sense to keep these methods in `Hook`(while adding them to `Repo`)  or take them out of `Hook`?